### ISSUE TITLE
Valve Index Controllers fail to be tracked

### ISSUE THREAD

**Issue by BlackCetha** [godotengine/godot#74832]
### Godot version

v4.0.stable.official.92bee43ad

### System information

Windows 10

### Issue description

I tried to follow the minimal [OpenXR tutorial](https://docs.godotengine.org/en/stable/tutorials/xr/setting_up_xr.html).
In order to see controller tracking working, I attached MeshInstance3D nodes with SphereMeshes to my XRController3D nodes.

I expected these nodes to move with my controllers, but they never did. The position of the XRController3D nodes stays at the origin.
I have a Valve Index Headset with Knuckles Controllers. Both the stable and beta version of SteamVR didn't work.
A friend of mine was able to follow the tutorial, but he used a Meta Quest 2 via PC Link. His controllers worked without further tweaking.

### Steps to reproduce

Follow the basic [OpenXR tutorial](https://docs.godotengine.org/en/stable/tutorials/xr/setting_up_xr.html).
I didn't adjust the position of nodes manually.

I downloaded Godot through Steam.

### Minimal reproduction project

[OpenXRTest.zip](https://github.com/godotengine/godot/files/10951469/OpenXRTest.zip)

**Comment by BlackCetha** [comment]
I also had a look at the SteamVR logs and noticed that it automatically generates a config file on every game launch.

While I don't really understand the format, it seems to me like the paths used for pose binding are different from the spec.
For example, the grip_pose is bound to `/user/hand/left/pose/grip`, when the [specification](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_valve_index_controller_profile) says it should be `/user/hand/left/input/grip/pose`.

You can have a look at the generated file here: https://gist.github.com/BlackCetha/b6fc158cc14bbafb28f5d94fefd99a28
SteamVR puts these at /Program Files (x86)/Steam/config/openxr.

**Comment by BastiaanOlij** [comment]
The config files in SteamVR are using SteamVRs internal path names which are different from OpenXRs so that isn't your problem.

I tested your test project on my system also with an Index and it works as it should.

Do you by any chance have an AMD graphics card? There is a known driver issue on AMD graphics cards that causes the sky to be drawn over everything else. This issue is fixed if you upgrade to the latest drivers from AMD.

**Comment by BlackCetha** [comment]
Thanks for your reply, but unfortunately my graphics card is from Nvidia.

I don't have a problem getting the game to render, even the headset tracking works out of the box. But I cannot get the XRController3D nodes to move with my controllers. Whatever setting I used, they stayed where I put them in the editor.

My friend also followed the OpenXR tutorial and send me his project files. As mentioned, it worked on his end with Oculus Touch Controllers. But when I loaded the same project, the controller nodes didn't move.

I've played other VR games in the meantime and have no issues with controller tracking.

**Comment by BlackCetha** [comment]
From what I can tell my game never receives XR_SESSION_STATE_VISIBLE or XR_SESSION_STATE_FOCUSED events.
Once again comparing to what I'm seeing to my friend on Oculus Link: He gets both those events when he puts on his headset. After these events are printed, several additional events for interaction profile updates are logged.

I have attached a log that I got by launching the editor with the verbose flag:
https://gist.github.com/BlackCetha/437a588afb899baa77d167fd59d88542
